README.md

```diff
@@ -802,66 +787,53 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   STATES = ['draft', 'open', 'closed']
   ~~~
 
-* Usage of trailing comma in multi-line collection literals is encouraged. It
-  makes diffs smaller and more meaningful.
+* Append a trailing comma in multi-line collection literals.
```

Comment:
the change is pretty hard from "it is encouraged" to "do it" maybe we could add a prefer or something. Unless ofc we want that ALL the time.

Reply:
Yes, and that's exactly why I'm changing the wording from a passive voice into the imperative tense. We shouldn't give space for doubt, the style guide should be as straightforward as possible.

Reply:
I realize the intention of the PR was different and that's my misunderstanding, sorry - carry on :D